monkestation/code/modules/cassettes/cassette_approval.dm

```diff
@@ -78,17 +78,27 @@ GLOBAL_LIST_INIT(cassette_reviews, list())
 /datum/cassette_review/Destroy(force)
 	. = ..()
```

Comment:
always better to put `return ..()` at the end of Destroy, instead of `. = ..()` at the beginning. not both, mind you